src/functTest/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainerFunctionalTest.groovy

```diff
@@ -75,13 +79,77 @@ class DockerCreateContainerFunctionalTest extends AbstractFunctionalTest {
         result.output.contains("nine=ten")
     }
 
-    static String containerUsage(String containerCreationTask) {
+    def "with autoRemove set, the container is removed after stopping"() {
+        given:
+        String containerCreationTask = """
+            task createContainer(type: DockerCreateContainer) {
+                dependsOn pullImage
+                targetImageId { pullImage.getImageId() }
+                autoRemove = true
+            }
+        """
+
+        String containerInspect = """
+            task inspectStoppedContainer(type: DockerInspectContainer) {
+                dependsOn stopContainer
+                targetContainerId { startContainer.getContainerId() }
+
+                onError = { err -> println 'RESULT: ' + err }
```

Comment:
looks like this should likely be `onError {` rather than the direct property assignment and that may also allow use of  a GString (which was giving issues)